dom/webidl/BrowserElement.webidl

```diff
@@ -30,40 +30,40 @@ BrowserElement implements BrowserElementPrivileged;
 interface BrowserElementCommon {
   [Throws,
    Pref="dom.mozBrowserFramesEnabled",
```

Comment:
This pref is also tested in `AudioChannelManager.webidl`, `BrowserElementAudioChannel.webidl`, and `BrowserElementProxy.webidl`. Not sure if you care about those cases though.